### PR TITLE
[Fix #13] Add datepicker options to event modal

### LINKED ISSUE #13: Date input field in "Add Event" modal is not formatted correctly
Current formatting of `date_field` is `mm/dd/yyyy`. It's better to change it to `dd/mm/yyyy`.

### PR DESCRIPTION
**Pull request by alahab2**
This change adds a "boostrap-datepicker" gem and lets a user pick a date using an interactive calendar.

**Note:** This change adds a gem, please run `bundle install'.

---

See screenshot below:

screencapture-localhost-3000-events-1476353280635

---

**Before submitting, check that:**

 - [x] You have written good* commit messages.
 - [x] You have squashed relevant commits together.
 - [x] You have ensured that RuboCop is passing.
 - [x] Your PR relates to one subject, with a clear title and
       description using complete, grammatically correct sentences.

---

If your change contains views, ensure that:

 - [x] You have included a screenshot of the new view.

---

*If you do not know what makes a good commit message, or why it
is important, please refer to these resources: [1][1], [2][2], [3][3].

[1]: https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message
[2]: http://chris.beams.io/posts/git-commit/
[3]: https://github.com/blog/1943-how-to-write-the-perfect-pull-request

- Install "bootstra-datepicker" gem
- Initialize datepicker and add it to events modal

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ gem "bootstrap", "~> 4.0.0.alpha4"
 gem "font-awesome-rails"
 
 gem "clearance"
+gem "bootstrap-datepicker-rails"
```
Comment (Drenmi) [inline Gemfile]: Let's group this with the other Bootstrap stuff in the Gemfile. 😀

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -15,3 +15,4 @@
 
  @import "bootstrap";
  @import "font-awesome";
+ @import "bootstrap-datepicker";
```
Comment (Drenmi) [inline app/assets/stylesheets/application.scss]: Let's nudge this up one line, next to the other Bootstrap stuff. 😀

File: app/views/events/_new.html.erb [hunk 3]
```diff
@@ -14,12 +14,12 @@
 
             <div class="form-group">
               <%= f.label :start_on, "Start date", class: "control-label" %>
-              <%= f.date_field :start_on, class: "form-control" %>
+              <%= f.text_field :start_on, class: "form-control datepicker", placeholder: "dd/mm/yyyy" %>
```
Comment (Drenmi) [inline app/views/events/_new.html.erb]: Let's [remove the placeholders](https://www.nngroup.com/articles/form-design-placeholders/). 😀

**Comment by alahab2** [comment]
Fixed! Interesting stuff about the placeholders, thanks!

**Pull request merged**